cli/command/inspect/inspector.go

```diff
@@ -110,6 +110,8 @@ func (i *TemplateInspector) tryRawInspectFallback(rawElement []byte) error {
 	buffer := new(bytes.Buffer)
 	rdr := bytes.NewReader(rawElement)
 	dec := json.NewDecoder(rdr)
+	// unmarshal a number into an interface{} as a Number instead of float64.
+	dec.UseNumber()
```

Comment:
One more nit:

This comment doesn't tell us much about why, it just repeats the docstring on `UseNumber()` (which should be easy for someone to look up).

I think we could either remove this comment, or make it specific to this usage:

> // Ensure numbers are always printed in the same format as they are received from the API

or something like that.